app/_locales/es/messages.json

```diff
@@ -1,10 +1,840 @@
 {
+  "accept": {
+    "message": "Aceptar"
+  },
+  "account": {
+    "message": "Cuenta"
+  },
+  "accountDetails": {
+    "message": "Detalles de la cuenta"
+  },
+  "accountName": {
+    "message": "Nombre de la cuenta"
+  },
+  "address": {
+    "message": "Direccion"
+  },
+  "addToken": {
+    "message": "Agregar Token"
+  },
+  "amount": {
+    "message": "Cantidad"
+  },
+  "amountPlusGas": {
+    "message": "Cantidad + Gas"
+  },
+  "appDescription": {
+    "message": "Extension del explorador usar Ethereum",
+    "description": "La descripcion de la aplicacion"
+  },
   "appName": {
     "message": "MetaMask",
-    "description": "The name of the application"
+    "description": "El nombre de la aplicacion"
   },
-  "appDescription": {
-    "message": "Administración de identidad en Ethereum",
-    "description": "The description of the application"
+  "attemptingConnect": {
+    "message": "Intentando conectar a la Blockchain"
+  },
+  "available": {
+    "message": "Disponible"
+  },
+  "back": {
+    "message": "Atras"
+  },
+  "balance": {
+    "message": "Balance:"
+  },
+  "balances": {
+    "message": "Tus balances"
+  },
+  "balanceIsInsufficientGas": {
+    "message": "Insuficiente balance para el gas total actual"
+  },
+  "beta": {
+    "message": "BETA"
+  },
+  "betweenMinAndMax": {
+    "message": "Debe ser mayor o igual a $1 y menor o igual a $2",
+    "description": "helper para ingresar hex como un ingreso decimal"
+  },
+  "borrowDharma": {
+    "message": "Pedir prestado con Dharma (Beta)"
+  },
+  "buy": {
+    "message": "Comprar"
+  },
+  "buyCoinbase": {
+    "message": "Comprar en Coinbase"
+  },
+  "buyCoinbaseExplainer": {
+    "message": "Coinbase es la forma mas popular en el mundo para comprar y vender bitcoin, ethereum, y litecoin."
+  },
+  "cancel": {
+    "message": "Cancelar"
+  },
+  "clickCopy": {
+    "message": "Click para copiar"
+  },
+  "confirm": {
+    "message": "Confirmar"
+  },
+  "continue": {
+    "message": "Continuar"
+  },
+  "confirmContract": {
+    "message": "Confirmar contracto"
+  },
+  "confirmPassword": {
+    "message": "Confirmar contraseña"
+  },
+  "enterPasswordConfirm": {
+    "message": "Ingresa tu password para confirmar"
+  },
+  "confirmTransaction": {
+    "message": "Confirmar transaccion "
+  },
+  "continueToCoinbase": {
+    "message": "Continuar a Coinbase"
+  },
+  "contractDeployment": {
+    "message": "Desplegar contrato"
+  },
+  "currentConversion": {
```

Comment:
I can't see this string here https://github.com/MetaMask/metamask-extension/blob/master/app/_locales/en/messages.json please @danfinlay confirm